server/src/main/resources/application.yml

```diff
@@ -14,6 +14,8 @@ micronaut:
       expire-after-write: 300
       heap:
         max-entries: 600
+    role-permission-cache:
+      expire-after-write: 86400
```

Comment:
This is pretty long...but, I guess why not? The service invalidates if we change things, so I guess I'm down. 🤷

Reply:
I think I just used the same value as the member cache.